webui/react/src/hooks/useModal/Model/useModalModelCreate.tsx

```diff
@@ -62,7 +61,6 @@ const DEFAULT_MODAL_STATE = {
   modelName: '',
   tags: [],
   visible: false,
-  workspaceId: 1,
```

Comment:
eventually i think we can switch to form hook instead of using useState here, but so far i just removed `workspaceId`. This avoid having default workspaceId